src/test/resources/alma-fix/990156060190206441.json

```diff
@@ -222,6 +226,7 @@
     "serialNumber" : "91 AVY 2",
     "currentLibrary" : "EUP",
     "currentLocation" : "EUP_BV",
+    "opacLink" : "https://onlinekatalog.bibliothek.fh-aachen.de/webOPACClient/start.do?Query=-1=%22%22",
```

Comment:
@blackwinter this is an example of the failing paste. I am not sure why.

Reply:
The title doesn't have a `zdbId`. You should wrap your `opacLink` lookups in `exists("xxxId")` (and preferably reverse the order, i.e. `hbzId` before `zdbId`).

Reply:
Thanks a lot! This worked!

Reply:
Please note that you also have to check if `hbzId` exists. Not all records have one.

Reply:
You are right, we also need to exclude hbzId when zdbId exists.